### ISSUE TITLE
macOS Disk Size

### ISSUE THREAD

**Issue by unixb0y** [KittyKatt/screenFetch#685]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] new distro request
```

**Bug report**
My disk usage is around 190GB, but shows as 11G / 500G.

```
> screenfetch -v
...
:: Finding current disk usage...found as '11G / 500G (4%%)'
...
```

If I execute the lines of the scrip that should be executed manually, it works fine though:
```
> totaldisk=$(df -H /System/Volumes/Data 2>/dev/null | tail -1)
> awk '{print $3}' <<< "${totaldisk}"
189G
```

**Comment by FrankKair** [comment]
**+1** I'm experiencing the same issue

**Comment by KittyKatt** [comment]
Hmmmm. Unsure why that's happening. I'll need to check it out locally and see what's happening. Thanks for the report!

**Comment by unixb0y** [comment]
@KittyKatt awesome thanks!

**Comment by vu1Art1st** [comment]
@unixb0y @FrankKair 
I recently encountered this problem and I don’t know if you still need a solution.
To solve the bug, My change was:
[Original code] `totaldisk=$(df -H / 2>/dev/null | tail -1)`
[Changed code] `totaldisk=$(df -H /System/Volumes/Data 2>/dev/null | tail -1)`

**Comment by unixb0y** [comment]
> @unixb0y @FrankKair I recently encountered this problem and I don’t know if you still need a solution. To solve the bug, My change was: [Original code] `totaldisk=$(df -H / 2>/dev/null | tail -1)` [Changed code] `totaldisk=$(df -H /System/Volumes/Data 2>/dev/null | tail -1)`

Awesome, this seems to work! I just realized that it's already part of `screenfetch-dev` in the main branch of this repo.
So I just copied all its contents into my screenfetch bash script.

Seems like a new release is due, no? Been a few years since the last one.

**Comment by vu1Art1st** [comment]
> > @unixb0y @FrankKair I recently encountered this problem and I don’t know if you still need a solution. To solve the bug, My change was: [Original code] `totaldisk=$(df -H / 2>/dev/null | tail -1)` [Changed code] `totaldisk=$(df -H /System/Volumes/Data 2>/dev/null | tail -1)`
> 
> Awesome, this seems to work! I just realized that it's already part of `screenfetch-dev` in the main branch of this repo. So I just copied all its contents into my screenfetch bash script.
> 
> Seems like a new release is due, no? Been a few years since the last one.

Yeah, it seems like a new release is due, and maybe you can take a look at issue #750? Those people also helped me fix a bug when I used it on Ventura 13.3.1 (Intel i9).

**Comment by unixb0y** [comment]
**FIXED IN MAIN BRANCH VERSION. NOT IN LATEST (3.9.1, 4 years old....) RELEASE THOUGH.**
